include/mbedtls/ssl.h

```diff
@@ -1956,6 +1965,27 @@ void mbedtls_ssl_conf_authmode( mbedtls_ssl_config *conf, int authmode );
 */
 void mbedtls_ssl_tls13_conf_early_data( mbedtls_ssl_config *conf,
                                         int early_data_enabled );
+#endif /* MBEDTLS_SSL_CLI_C */
+
+#if defined(MBEDTLS_SSL_SRV_C)
+/**
+ * \brief Set the max_early_data_size parameter.
+ *
+ * \param[in] conf                  The SSL configuration to use.
+ * \param[in] max_early_data_size   The maximum amount of 0-RTT data.
+ *                                  - 0 Disable 0-RTT feature.
```

Comment:
```suggestion
 *                                  - 0 Disable early data feature.
```

Reply:
Prefer `0-RTT` .

Reply:
For this description, `0-RTT` is better than `early data` . `0-RTT` is the word that RFC used for early app data. And `early data` might confuse with `early data extension`.